x-pack/plugins/security_solution/cypress/integration/detection_rules/event_correlation_rule.spec.ts

```diff
@@ -59,151 +55,145 @@ import {
   waitForAlertsToPopulate,
   waitForTheRuleToBeExecuted,
 } from '../../tasks/create_new_rule';
-import { loginAndWaitForPageWithoutDateRange } from '../../tasks/login';
+import { login, visit } from '../../tasks/login';
 
 import { RULE_CREATION } from '../../urls/navigation';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
 
-describe('Detection rules, EQL', () => {
-  const expectedUrls = getEqlRule().referenceUrls.join('');
-  const expectedFalsePositives = getEqlRule().falsePositivesExamples.join('');
-  const expectedTags = getEqlRule().tags.join('');
-  const expectedMitre = formatMitreAttackDescription(getEqlRule().mitre);
-  const expectedNumberOfRules = 1;
-  const expectedNumberOfAlerts = '7 alerts';
-
-  beforeEach(() => {
+describe('EQL rules', () => {
+  before(() => {
     cleanKibana();
-    createTimeline(getEqlRule().timeline).then((response) => {
-      cy.wrap({
-        ...getEqlRule(),
-        timeline: {
-          ...getEqlRule().timeline,
-          id: response.body.data.persistTimeline.timeline.savedObjectId,
-        },
-      }).as('rule');
-    });
+    login();
+    deleteAlertsAndRules();
   });
+  describe('Detection rules, EQL', () => {
+    const expectedUrls = getEqlRule().referenceUrls.join('');
+    const expectedFalsePositives = getEqlRule().falsePositivesExamples.join('');
+    const expectedTags = getEqlRule().tags.join('');
+    const expectedMitre = formatMitreAttackDescription(getEqlRule().mitre);
+    const expectedNumberOfRules = 1;
+    const expectedNumberOfAlerts = '1 alert';
+
+    beforeEach(() => {
+      createTimeline(getEqlRule().timeline).then((response) => {
+        cy.wrap({
+          ...getEqlRule(),
+          timeline: {
+            ...getEqlRule().timeline,
+            id: response.body.data.persistTimeline.timeline.savedObjectId,
+          },
+        }).as('rule');
+      });
+    });
 
-  it('Creates and enables a new EQL rule', function () {
-    loginAndWaitForPageWithoutDateRange(RULE_CREATION);
-    selectEqlRuleType();
-    fillDefineEqlRuleAndContinue(this.rule);
-    fillAboutRuleAndContinue(this.rule);
-    fillScheduleRuleAndContinue(this.rule);
-    createAndEnableRule();
-
-    cy.get(CUSTOM_RULES_BTN).should('have.text', 'Custom rules (1)');
+    it('Creates and enables a new EQL rule', function () {
```

Comment:
nit: At some point we really should enforce a linter rule of using just fat arrow over `function () {`, you don't have to change this here, just an observation.